### PR TITLE
[CDAP-7864] Fixes css bug and places close btn in cask mkt

### PR DESCRIPTION
**Pull request by PGuimarais**
screen shot 2016-12-20 at 5 09 15 pm

- Places 'X' button in modal header for dismissal 
- Fixes sidebar css bug

JIRA:
https://issues.cask.co/browse/CDAP-7864

### CODE REVIEW

File: cdap-ui/app/cdap/components/PlusButtonModal/PlusButtonModal.less [hunk 1]
```diff
@@ -53,6 +53,16 @@
           margin-right: 2px;
         }
 
+        .modal-close-btn {
+          width: 51px;
+          height: 100%;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/PlusButtonModal/PlusButtonModal.less]: Neither `height` nor `margin-right`  is needed here. `vertical-align: middle` is what needed here.

File: cdap-ui/app/directives/my-global-navbar/my-global-navbar.less [hunk 2]
```diff
@@ -155,6 +155,9 @@ my-global-navbar {
         padding-left: 20px;
         a {
           padding-right: 20px;
+          &.top-product {
+            padding-right: 0;
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/my-global-navbar/my-global-navbar.less]: Why is this needed? Not sure if this change is related to this PR

File: cdap-ui/app/cdap/components/PlusButtonModal/PlusButtonModal.less [hunk 3]
```diff
@@ -53,6 +53,15 @@
           margin-right: 2px;
         }
 
+        .modal-close-btn {
+          display: inline-flex;
+          cursor: pointer;
+          justify-content: center;
+          align-items: center;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/PlusButtonModal/PlusButtonModal.less]: can we use text-center in this?

**Comment by elfenheart** [comment]
👍  LGTM
